src/mococrw/openssl_wrap.h

```diff
@@ -1374,5 +1374,7 @@ SSL_EC_KEY_Ptr _EC_KEY_oct2key(int nid, const std::vector<uint8_t> &buf);
 void _EVP_PKEY_set1_EC_KEY(EVP_PKEY *pkey, const EC_KEY *key);
 
 std::vector<uint8_t> _EC_KEY_key2buf(const EVP_PKEY* evp, point_conversion_form_t form);
+std::vector<uint8_t> _ECDH_compute_key(const EVP_PKEY *pub_key, const EVP_PKEY *ecdh,
```

Comment:
Can we make the parameter names more expressive? I do get what the `pub_key` is, but I'm not sure why the private key is named `ecdh`.

I'd propose to use the nomenclature used by the OpenSSL code, which is `key` for the private key and `peerkey` for the public key of the other peer.

Reply:
Done